tests/integration/ceph_base_object_test.go

```diff
@@ -190,6 +192,26 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	require.Nil(s.T(), delobjErr)
 	logger.Infof("Objects deleted on bucket successfully")
 
+	// A bug exists in older versions of lib-bucket-provisioner that will revert a bucket and claim
+	// back to "Pending" phase after being created and initially "Bound" by looping infinitely in
+	// the bucket provision/creation loop. Verify that the OBC is "Bound" and stays that way. We
+	// need to verify this before the OBC is deleted, but we can avoid wasting extra time sleeping
+	// in a wait loop by completing all tests before this regression check.
+	logger.Infof("Regression check: Verify bucket does not revert to Pending phase")
+	// The OBC reconcile loop usually runs again in 30-90 seconds after being created due to the OBC
+	// resource being modified to refer to its OB.
+	timeToWaitForReversion := 2 * time.Minute
+	timeToCheckForReversion := timeBucketCreateVerified.Add(timeToWaitForReversion)
+	for time.Now().Before(timeToCheckForReversion) {
```

Comment:
Is this `for` loop the same as a single 2 minute sleep? Why not just do this? 
`time.Sleep(2 * time.Minute)`
Actually, we really should not have a two minute sleep in the test. That's pretty expensive when the whole suite is less than 15 minutes. I would suggest we not test for this revert bug in the integration tests.

Reply:
I didn't want to just blindly wait 2 minutes, so I check every 5 seconds if it's been 2 minutes since the OBC was created `timeBucketCreateVerified`.

Reply:
The best thing to do might be to make the object and bucket its own test outside of the smoke suite. Maybe a separate PR? I really think we should do a better job of adding regression tests into our integration.

Reply:
This is where we really do need a "daily" test suite that will run regression tests that may take a lot longer to run. For development we really need to keep the test suites shorter though.

Ideally we would add a test to prevent every regression that we find. In reality, it's frequently just not feasible to do that, so we have to strike a balance based on the cost of creating the test.

Reply:
I'm open to adding this regression in a separate PR with the ability to enable it only with a `REGRESSION_TEST` env var or something, but I really don't want to just drop the responsibility of writing regression tests. Maybe I can work with Seb to get another GH action to run regressions daily and on release builds?

Reply:
Yes, a separate PR sounds good for the tests so we can this fix into the release.